evaporate.js

```diff
@@ -14,25 +14,27 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
 
 /***************************************************************************************************
 *                                                                                                  *
-*  version 0.0.2                                                                                  *
-*                                                                                                  *
-*  TODO:                                                                                           *
-*       calculate MD5s and send with PUTs                                                          *
-*       post eTags to application server to allow resumability after client-side crash/restart      *
-*                                                                                                  *
+*  version 1.0.0                                                                                   *
 *                                                                                                  *
 ***************************************************************************************************/
 
 (function() {
-
-  var Evaporate = function(config){
-
-     this.supported = !((typeof(File)=='undefined') ||
-        (typeof(Blob)=='undefined') ||
-        !(!!Blob.prototype.webkitSlice || !!Blob.prototype.mozSlice || Blob.prototype.slice) ||
+   var FAR_FUTURE = new Date('2060-10-22');
+
+  var Evaporate = function (config) {
+
+     this.supported = !(
+        typeof File === 'undefined' ||
+        typeof Blob === 'undefined' ||
+        typeof (
+           Blob.prototype.webkitSlice ||
+           Blob.prototype.mozSlice ||
+           Blob.prototype.slice ||
+           (config.computeContentMd5 && Blob.prototype.readAsBinaryString) === 'undefined') ||
```

Comment:
I think this test is wrong. `readAsBinaryString` is method on FileReader, not Blob (currently the latest Chrome fails this test), and it's allegedly been replaced by `readAsArrayBuffer` in the spec